packages/manager/apps/shell/vite.config.js

```diff
@@ -33,6 +31,17 @@ function viteOvhDevServerPlugin() {
       } else {
         app.use('/app', express.static(appDistPath));
       }
+
+      app.get('/auth', sso.auth.bind(sso));
+      app.get('/auth/check', sso.checkAuth.bind(sso));
+
+      if (env.local2API) {
```

Comment:
`env.local2API` is a boolean that tells us if 2API are local or not?

Reply:
it checks the presence of the env var `local2API` no matter its value.

This command will enable `local2API` proxy:
```shell
$ local2API=1 yarn dev
```

This one won't:
```shell
$ yarn dev
```